ecogvis/functions/nwb_copy_file.py

```diff
@@ -266,10 +266,10 @@ def copy_obj(obj_old, nwb_old, nwb_new):
 
     # ElectricalSeries --------------------------------------------------------
     if type(obj_old) is ElectricalSeries:
-        nChannels = len(obj_old.electrodes.table['x'].data)
-        elecs_region = nwb_new.electrodes.create_region(
+        region = list(obj_old.electrodes.to_dataframe().index)
```

Comment:
Here you are converting to a dataframe, which will work but is unnecessarily costly if all you want is the ids in the region. It is more efficient to first get the ids from the table, and then index them by the data field of the `ElectricalSeries.electrodes.data`. See the eventual fix for similar code in SpikeInterface:

https://github.com/SpikeInterface/spikeextractors/pull/401/files